content/docs/hooks-overview.md

```diff
@@ -201,9 +201,9 @@ function useFriendStatus(friendID) {
 }
 ```
 
-It takes `friendID` as an argument, and returns whether our friend is online.
+यह एक तर्क के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है। यह एक आर्गुमेंट के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है।
```

Comment:
`लौटता` => `रिटर्न करता`

Reply:
you missed this

Reply:
I don't see the old version, I changed it before. Can you check again?

Reply:
You can check this line, its still there 

https://github.com/reactjs/hi.reactjs.org/pull/58/files#diff-9bda51acb98cffd656661def0255b05bR204

Maybe you forgot to push your changes?

Reply:
@arshadkazmi42 
Yeah, that was the case. Pushed it now. Thanks for noticing.